src/components/common/modal/index.js

```diff
@@ -0,0 +1,21 @@
+import React, {PureComponent} from 'react';
+import PropTypes from 'prop-types';
+import Button from "../button/index";
```

Comment:
`index` not needed here, import from directory `import Button from '../button';`